pkg/lang/javascript/aws_runtime/.gitignore

```diff
@@ -1 +1,12 @@
 /node_modules/
+dispatcher_fargate.js.tmpl
+dispatcher_lambda.js.tmpl
+emitter.js.tmpl
+fs.js.tmpl
+keyvalue.js.tmpl
+orm.js.tmpl
+proxy_eks.js.tmpl
+proxy_fargate.js.tmpl
+redis_cluster.js.tmpl
+redis_node.js.tmpl
+secret.js.tmpl
```

Comment:
why ignore checking these in? we dont have .js files for all of them so we need some of these checked in

Reply:
Which ones don't we have `.ts` files for? I actually even create the changes to this file manually; they're all from the bash script's changes to add the generated files to this ignore.

Reply:
oh i guess this list doesnt include the ones which we dont have ts files for